app/controllers/tags_controller.rb

```diff
@@ -112,7 +116,7 @@ def feed
         raise ActiveRecord::RecordNotFound, "Couldn't find tag with id '#{params[:id]}'"
       end
       tag = tag.merger if !tag.canonical? && tag.merger
-      # Temp for testing
+      # F/F is currently a special case...
```

Comment:
My understanding is that it isn't a change for test but adding the ability for F/F to have a feed when it wouldn't normally ?